domains/transfer/Transfer.API/API.UnitTests/Transfer/Api/Converters/Base64ConverterTests.cs

```diff
@@ -30,7 +30,7 @@ public void Base64Converter_ShouldBeAbleToConvertBackAfterConversion()
     }
 
     [Theory]
-    [InlineData(null)]
```

Comment:
Why not null anymore?

Reply:
Warning from xunit linter. will find alternative solution

Reply:
I fixed it here:

https://github.com/Energinet-DataHub/energy-origin/pull/558/commits/aa903c41a8672c0e123784cf89cd7bb231b43fb1